src/cpu.rs

```diff
@@ -3327,7 +3327,7 @@ impl CPU {
     fn segment(&self, seg: Segment) -> u16 {
         match seg {
             Segment::CS() |
-            Segment::Default() => self.sreg16[CS],
+            Segment::Default() => self.sreg16[DS],
```

Comment:
This is wrong, since it now also resolves Segment::CS()  to self.sreg16[DS]

Reply:
you right, give me a second